development/components/form/_index.md

```diff
@@ -1,8 +1,276 @@
 ---
-title: The Form component
+title: Forms with Symfony
 menuTitle: Form
+useMermaid: true
 ---
 
-# The Form component
+# Forms with Symfony
 
-{{% children /%}}
+## Introduction
+
+With Symfony, `Forms` are built, rendered and handled in separate layers. Several concepts are involved, such as form building, providing data to forms, validating data, persisting data submited by forms, rendering forms, ...
+
+- [`Controllers`](#controllers) - Handles all logic for the `Form`
+- [`FormBuilder`](#form-builder) - Builds the `Form`, and provide data to the `Form`
+    - [`FormDataProvider`](#form-data-provider) - Provides data to the `Form`
+- [`Form`](#form-types) - Defines the `Form` and its content
+- [`Form Theme`](#form-theme) - Allows to customise the style of the `Form`
+- `Form Templates` - Allows to render the `Form` with `Twig`
+- [`FormHandler`](#form-handler) - Handles the submited `Form`, provides data to the `FormDataHandler`, and handles `FormHandlerResult`
+    - [`FormDataHandler`](#form-handler) - Persists the data to `Database`
+
+<div class='mermaid'>
+graph
+    AA(Http Request)-.->A
+    subgraph Core
+        subgraph Admin Theme
+            K(Form Theme)
+        end
+        subgraph Render form
+            A(Controller)-->O(FormBuilder)
+            O-->|1|M(FormDataProvider)
+            M-->|2|O
+            O-->|3|P(Form)
+            P-->F(Rendered Form)
+            K-->F
+            A-->Q(Form Templates)
+            Q-->F
+        end
+        subgraph Handle form
+            F -- Submit -->G(Controller)
+            G-->H(FormHandler)
+            H-->|Persist|Z(FormDataHandler)
+        end
+    end
+    G-.->ZZ(Redirect)
+    style F fill:#f9f,stroke:#333,stroke-width:2px
+    style Core fill:#fbfbea
+</div>
+
+### Controllers
+
+`Controllers` in the back office receives http requests, and when handling requests for creating or editing an entity, they instanciate and uses several concepts.
```

Comment:
```suggestion
`Controllers` in the back office receive HTTP requests. When handling requests for creating or editing an entity, they instantiate and utilize several concepts.
```

Reply:
I'm not sure about adding `Highlights` to the names that are not names of the classes, this is why I decided to get rid of them later :D